docs/README.md

```diff
@@ -24,5 +144,42 @@ The full architecture and vision of the Crossplane project is described in depth
   * [Running Resources](running-resources.md)
   * [Troubleshooting](troubleshoot.md)
 * [Concepts](concepts.md)
+* [API Reference](api.md)
 * [FAQs](faqs.md)
 * [Contributing](contributing.md)
+
+### Next Steps
+Use [stack-gcp][stack-gcp], [stack-aws][stack-aws], and [stack-azure][stack-azure] in the the following guides:
+* [Crossplane Service Guide][services-user-guide] - upgrade an existing Kubernetes cluster
```

Comment:
Upgrade it to do what? I would suggest that we use something other than `upgrade`, since the normal meaning for `upgrade` in this context would be to increment the version of Kubernetes which is running on the cluster. For example, maybe we could say something more like `add Crossplane to an existing Kubernetes cluster, and use them together to manage service provisioning from kubectl`